homeassistant/components/opnsense/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "domain": "opnsense",
+  "name": "OPNSense",
+  "documentation": "https://www.home-assistant.io/components/opnsense",
+  "requirements": [
+    "pyopnsense==0.2.0"
+  ],
+  "dependencies": [],
+  "codeowners": []
```

Comment:
Please add yourself as code owner.

Reply:
Oh, I just assumed I didn't need to because I'm not listed as a codeowner on any of the components I've contributed in the past. Figured it was a requirement. Fixed in: https://github.com/home-assistant/home-assistant/pull/26834/commits/914fce5e613221f69c8fcb7edbcf20a97b0c418d